home/web/client/actions/dapps.js

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions';
+
+export const error = createAction('error dapps');
```

Comment:
yeah, but I think it's part of fetching dapps, right?

We should keep it consistent for async actions for instance:
```
fetchDappsStarted
fetchDappsSuccess
fetchDappsErrors
```